microdnf/commands/install/install.cpp

```diff
@@ -102,12 +102,23 @@ void CmdInstall::run(Context & ctx) {
 
     download_packages(goal, nullptr);
 
+    std::cout << std::endl;
+
+    libdnf::rpm::Transaction rpm_transaction(ctx.base);
```

Comment:
Following lines should be in future in `libdnf::cli` or libdnf::base as a method. What about to move the code to microdnf/contect? Then it can be shared by multiple commands.

Reply:
I thought about it. But this code is temporary. It will change significantly in the future. I don't want to invest time moving this temporary code.